src/context.ts

```diff
@@ -262,6 +268,23 @@ export class Context<E extends WebhookPayloadWithRepository = any>
     ) as unknown) as T;
   }
 
+  /**
+   * config without fallback when file does not exist
+   * @see config
+   *
+   * @param fileName - Name of the YAML file in the `.github` directory
+   * @param defaultConfig - An object of default config options
+   * @param deepMergeOptions - Controls merging configs (from the [deepmerge](https://github.com/TehShrike/deepmerge) module)
+   * @return Configuration object read from the file
+   */
+  public configStrict<T>(
+    fileName: string,
+    defaultConfig?: T,
+    deepMergeOptions?: MergeOptions
+  ): Promise<T | null> {
+    return this.config(fileName, defaultConfig, deepMergeOptions, false);
+  }
+
```

Comment:
I would prefer not to add another API method. In 3 years I think you are the first who asked for that feature, I don't think it's common enough to grant a dedicated method.

Reply:
Yep that is fair - the method was more stretch :) Happy with just option in the current method.

As for no one asked - If I understand it correctly, this behaviour was changed when the `probot-config` was integrated into probot in https://github.com/probot/probot/commit/3f0491f63b854de5372e8bed0017b87c8762297e#diff-a7af09f7fa8732629ff9579258b5584b

Before that the fallback to the base was not there.